theia-full-docker/Dockerfile

```diff
@@ -249,12 +295,22 @@ ENV DART_VERSION 2.9.0
 
 RUN \
   apt-get update && apt-get install --no-install-recommends -y -q gnupg2 curl git ca-certificates apt-transport-https openssh-client && \
+  apt-get clean && \
```

Comment:
this cleanup block is not needed - we have one in the end of this `RUN` command, starting line 308.

Reply:
Good catch, I'll remove it.